Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/client/clientcmd/loader.go

```diff
@@ -90,8 +93,10 @@ func (rules *ClientConfigLoadingRules) Load() (*clientcmdapi.Config, error) {
 		if err := mergeConfigWithFile(mapConfig, file); err != nil {
 			errlist = append(errlist, err)
 		}
-		if err := ResolveLocalPaths(file, mapConfig); err != nil {
-			errlist = append(errlist, err)
+		if !rules.DoNotResolvePaths {
```

Comment:
If you really want to keep the `DoNotResolvePaths` I'd suggest a `ResolvePaths()` method for clarity.